data-processing/README.md

```diff
@@ -142,27 +142,27 @@ If you're just testing out the pipeline, consider making an
 Then run the following command to start off the pipeline:
 
 ```bash
-python scripts/process.py fetch-arxiv-sources arxiv_ids.txt
+python scripts/run_pipeline.py fetch-arxiv-sources arxiv_ids.txt
```

Comment:
So this doesn't work for me...
```
root@180a951ef375:/data-processing# echo 0801.4750 > arxiv_ids.txt
root@180a951ef375:/data-processing# 
root@180a951ef375:/data-processing# python scripts/run_pipeline.py fetch-arxiv-sources arxiv_ids.txt
wandb: WARNING W&B installed but not logged in.  Run `wandb login` or set the WANDB_API_KEY env variable.
usage: run_pipeline.py [-h] [-v] [--config CONFIG] [--log-prefix LOG_PREFIX]
                       [--entities {citations,sentences,symbols,definitions,equations,glossary-terms} [{citations,sentences,symbols,definitions,equations,glossary-terms} ...]]
                       [--commands {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,raster-pages,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-sentences,locate-bounding-boxes-for-sentences,upload-sentences,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions} [{fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,raster-pages,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-sentences,locate-bounding-boxes-for-sentences,upload-sentences,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions} ...]]
                       [--start {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,raster-pages,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-sentences,locate-bounding-boxes-for-sentences,upload-sentences,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions}]
                       [--end {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,raster-pages,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-sentences,locate-bounding-boxes-for-sentences,upload-sentences,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions}]
                       [--arxiv-ids ARXIV_IDS [ARXIV_IDS ...]]
                       [--arxiv-ids-file ARXIV_IDS_FILE] [--days DAYS]
                       [--s3-job-spec S3_JOB_SPEC] [--source {arxiv,s3}]
                       [--s3-arxiv-sources-bucket S3_ARXIV_SOURCES_BUCKET]
                       [--entity-batch-size ENTITY_BATCH_SIZE]
                       [--keep-intermediate-files] [--max-papers MAX_PAPERS]
                       [--one-paper-at-a-time] [--keep-paper-data]
                       [--store-results] [--store-log]
                       [--s3-output-bucket S3_OUTPUT_BUCKET]
                       [--notify-emails NOTIFY_EMAILS [NOTIFY_EMAILS ...]]
                       [--notify-bcc NOTIFY_BCC [NOTIFY_BCC ...]]
                       [--extraction-only] [--dry-run]
                       [--database-schema DATABASE_SCHEMA]
                       [--database-create-tables]
                       [--data-version DATA_VERSION]
run_pipeline.py: error: unrecognized arguments: fetch-arxiv-sources arxiv_ids.txt
```
Though something like this does:
```
root@9e48011a4466:/data-processing# python scripts/run_pipeline.py   --one-paper-at-a-time   --source arxiv   --arxiv-ids 1903.09673   --commands fetch-arxiv-sources -v
wandb: WARNING W&B installed but not logged in.  Run `wandb login` or set the WANDB_API_KEY env variable.
2021-03-31 15:37:44,341 [DEBUG]: Assembling the list of commands to be run.
2021-03-31 15:37:44,342 [DEBUG]: The following commands will be run, in this order: ['fetch-arxiv-sources']
2021-03-31 15:37:44,343 [INFO]: Running pipeline for paper 1903.09673
...
```
Though I guess the previous version wouldn't work for me either because I don't see a `scripts/process.py` anymore...

Reply:
Yeah `scripts/process.py` doesn't exist anymore and I was too lazy to really fix it so I just change it to `run_pipeline.py`. I'll update the instruction properly :p